web/packages/teleport/src/Discover/ConnectMyComputer/TestConnection/TestConnection.story.tsx

```diff
@@ -74,14 +112,22 @@ export const MultipleLogins = () => {
 
 MultipleLogins.parameters = {
   msw: {
-    handlers: [
-      rest.post(cfg.api.webRenewTokenPath, (req, res, ctx) =>
-        res(ctx.json({}))
-      ),
-      rest.get(cfg.api.connectMyComputerLoginsPath, (req, res, ctx) =>
-        res(ctx.json({ logins: ['foo', 'bar', 'baz'] }))
-      ),
-    ],
+    handlers: {
+      connectMyComputerLogins: [
+        rest.get(cfg.api.connectMyComputerLoginsPath, (req, res, ctx) =>
+          res(
+            ctx.json({
+              logins: [
+                'foo',
+                'bar',
+                'baz',
+                'czesława_maria_de_domo_cieślak_primo_voto_gospodarek_secundo_voto_kowalczyk',
```

Comment:
Server/TestConnection uses a shorter login:

https://github.com/gravitational/teleport/blob/170464fb5651f8bdfcaf4a3582bc5575d3a1b52f/web/packages/teleport/src/Discover/Server/TestConnection/TestConnection.story.tsx#L36-L39

and the width of the login select is set to be just wide enough to contain it, but it fails with the full name of Violetta Villas. 😭

<img width="850" alt="login-select-with-long-values" src="https://github.com/gravitational/teleport/assets/27113/b3f02940-20f5-43e8-8d97-e594cdf1101f">

For now I'm just not going to try to fix what's broken as I couldn't quickly figure out how to make react-select work in this scenario.